src/TEMPORARY_GLOBAL_FLAGS.ts

```diff
@@ -1,4 +1,4 @@
 // NATODO delete this file and remove all references when we are ready to release non-admin support
 // (will need to manually revert changes from https://github.com/konveyor/mig-ui/pull/966)
 
-export const NON_ADMIN_ENABLED = false;
+export const NON_ADMIN_ENABLED = true;
```

Comment:
We should leave this set to false and just flip it to true as an uncommitted change while testing this stuff, so we don't accidentally flip it to true on master

Reply:
Ah! yeah i keep accidentally committing this.

Reply:
Ended up just locally ignoring that file with: 

git update-index --skip-worktree src/TEMPORARY_GLOBAL_FLAGS.ts

Reply:
Ah! Didn't know you could do that, nice